packages/create-sitecore-jss/src/templates/nextjs/src/lib/graphql-client-factory/utils.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  GraphQLRequestClientFactoryConfig,
+  getEdgeProxyContentUrl,
+  GraphQLRequestClient,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+import { JssConfig } from 'scripts/config';
```

Comment:
Feels strange that we have the type in 'scripts/config'. Wonder if we should move this type somewhere else? Just an observation. We could potentially use this type in more places (wherever we are importing 'temp/config').

Reply:
Good point 👍 
Added _lib/config_ file for now